addons/website_slides_survey/views/website_slides_templates.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data>
+        <template id="slide_detail_view_inherit_survey" inherit_id="website_slides.slide_detail_view">
+            <xpath expr="//div[hasclass('o_w_slides_slide_detail_container')]/div[hasclass('row')]" position="before">
+                <t t-if="not test_entry" t-set="begin_certification_label" t-value="'Begin certification'" />
+                <t t-if="test_entry" t-set="begin_certification_label" t-value="'Test certification'" />
+                <div t-if="slide.slide_type == 'certification' and not quizz_passed" class="mb8">
+                    <a role="button"
+                        class="btn btn-primary btn-lg"
+                        t-att-href="certification_url">
+                        <i class="fa fa-fw fa-graduation-cap" role="img" t-att-aria-label="begin_certification_label" t-att-title="begin_certification_label"/>
+                        <t t-esc="begin_certification_label" />
+                    </a>
+                </div>
+                <div t-if="slide.slide_type == 'certification' and quizz_passed" class="mb8">
+                    <a role="button"
+                        t-att-data-survey_id="survey_id"
+                        class="btn btn-primary btn-lg o_wslides_survey_download_certification"
+                        href="#">
+                        <i class="fa fa-fw fa-trophy" role="img" aria-label="Download certification" title="Download certification"/>
+                        Download certification
+                    </a>
+                </div>
+            </xpath>
+        </template>
+
+        <template id="slides_channel_header_inherit_survey" inherit_id="website_slides.slides_channel_header">
```

Comment:
Should not be necessary I think with shiny '%ss' mechanism.

Reply:
Praise the '%ss'